CharacterCreation/02_Classes.txt

```diff
@@ -187,11 +187,10 @@ you're not busy fixing what other people broke, you get to work on making new
 stuff. Which some knucklehead will probably, eventually, break too.
 
 Class Primary Attribute: Intelligence
-
-Class Feats: 
-=======
 Starting Wealth: $2580 + 100*Luck
 
+== Class Feats: ==
+
```

Comment:
This still doesn't go here. The class feats subsection heading exists in its proper spot at line 227, consistent with all other classes. The class passives are not class feats.